R/impact_central.R

```diff
@@ -202,3 +196,106 @@ determine_vaccine_delivery <- function(meta1){
 #
 #
 # }
+
+#' Calculate impact by year of vaccination country perspective
+#'
+#' This will calculate the impact by year of vaccination by country and
+#' burden outcome for a single disease and vaccine.
+#'
+#' This can take data either by vaccination year and age at vaccination or
+#' by birth cohort year.
+#'
+#' @param raw_impact Data frame of raw impact data this needs to have
+#' columns country, value, burden_outcome and either year & age or birth_cohort
+#' @param fvps Data frame of fully vaccination person data with columns
+#' country, fvps and either year & age or birth_cohort
+#' @param activity_type `routine` or `campaign` activity type
+#' @param vaccination_years Years of vaccination of interest
+#'
+#' @return Impact ratio by country and burden outcome
+#' @export
```

Comment:
Really it would be ideal to have some examples here - I wonder if we have any representative data we can pull from the DB and store in the `inst` dir? I can then use it here in example

Reply:
I am not sure how sensible it is to pull real data into the package. If we want, maybe we have to keep country names anonymous, and downscale numbers by a random factor?